src/login/login_auth.cpp

```diff
@@ -107,7 +109,9 @@ int32 login_parse(int32 fd)
             const char* fmtQuery = "SELECT accounts.id,accounts.status \
 									FROM accounts \
 									WHERE accounts.login = '%s' AND accounts.password = PASSWORD('%s')";
-            int32 ret = Sql_Query(SqlHandle, fmtQuery, name.c_str(), password.c_str());
+            Sql_EscapeString(SqlHandle, escaped_name, name.c_str());
```

Comment:
Does this function assume output buffer is of same length as input buffer? Sounds bit unsafe.
I would at least stick assert before to make sure there's no buffer overflow.

Reply:
it assumes the output buffer is at least 2 times plus 1 bigger than the input (which is what I was talking about earlier)